### PR TITLE
Responsiveness Fix

### LINKED ISSUE #66: Improve  the Responsiveness of the website https://bub2.toolforge.org/
1. Correct the position of the Enter E-mail input field under the Upload Tab

2. Correct the position of the Card Section under the Queues Tab.

Please assign me this task

### PR DESCRIPTION
**Pull request by Xtremilicious**
Fixes #66 

1. Fixed the Footer overlapping with the Email field in smaller devices.
2. Made the Stats page responsive for mobile phones.

<div style="display: flex">
ScreenShot_20200323202817

ScreenShot_20200323202849
</div>

### CODE REVIEW

**Comment by Xtremilicious** [comment]
@coderwassananmol @ankitjena @rkpattnaik780 @Parikshit-Hooda Kindly review :)

File: components/Books.js [hunk 1]
```diff
@@ -274,9 +274,20 @@ export default class Books extends React.Component {
...
           </div>
-          <button className="btn btn-primary" type="submit">
+          <div>
+          <style jsx>
+              {
+                `
+              padding-top: 3vh;
+              padding-bottom: 3vh;
+              `
+              }
```
Comment (ankitjena) [inline components/Books.js]: Indent these

File: pages/stats.js [hunk 2]
```diff
@@ -1,66 +1,74 @@
...
-                </div>
-            </div>
-            <UploadedItems />
-        <Footer />
+const Stats = props => (
+  <div>
+    <Header page="stats" />
+    <style jsx>
+      {`
+        .card-container {
```
Comment (ankitjena) [inline pages/stats.js]: A bit extra padding below the stats card would look much nicer

Comment (Xtremilicious) [reply pages/stats.js]: Fixed

File: components/Books.js (hunk 1 above)
Comment (Xtremilicious) [reply components/Books.js]: Fixed

**Comment by Xtremilicious** [comment]
Kindly review and merge :)

**Comment by Xtremilicious** [comment]
@coderwassananmol kindly merge so that I can get started on #78

**Comment by Xtremilicious** [comment]
@rkpattnaik780 @Parikshit-Hooda  Kindly add he gssoc2020 label as well, it's not counting the points. Thanks :)

**Comment by Xtremilicious** [comment]
@coderwassananmol @ankitjena Kindly add the gssoc2020 label

**Pull request merged**